zipline/finance/blotter.py

```diff
@@ -179,64 +185,62 @@ def process_benchmark(self, benchmark_event):
         return
         yield
 
-    def process_trade(self, trade_event):
+    def process_open_orders(self):
```

Comment:
My strong preference would be for the blotter to not know anything about the current simulation time.  If it's just the order methods that need the `dt`, that's easily solved by passing `current_dt` to those as well.